src/Materials/Background/backgroundMaterial.ts

```diff
@@ -123,6 +123,7 @@ class BackgroundMaterialDefines extends MaterialDefines implements IImageProcess
     public SAMPLER3DGREENDEPTH = false;
     public SAMPLER3DBGRMAP = false;
     public IMAGEPROCESSINGPOSTPROCESS = false;
+    public CLAMPFINALCOLOR = true;

```

Comment:
As true is not the default value in rebuild, I think you either need to rename to a negative name like UNCLAMPFINALCOLOR = false, or override the reset to force the value back to true in all the defines objects IIRC.

Reply:
UNCLAMPFINALCOLOR sounded wrong...
But I agree that default false would be better.

Let's see what I can do

Reply:
CLAMPFINALCOLOR => SKIPFINALCOLORCLAMP